library/src/main/java/com/pokegoapi/api/pokemon/EvolutionInfo.java

```diff
@@ -419,6 +419,24 @@
 	}
 
 	/**
+	 * Get evolutions
+	 *
+	 * @param pokemonId pokemon id
+	 * @return ordered evolutions
+	 */
+	public static List<PokemonId> getEvolutions(PokemonId pokemonId) {
```

Comment:
Just because I still tend to forget about java's Arrays utility class... :shame: